src/main/scala/io/iohk/ethereum/utils/VersionInfo.scala

```diff
@@ -0,0 +1,48 @@
+package io.iohk.ethereum.utils
+
+object VersionInfo {
+
+  /** Produce a node name that get sent in `WireProtocol.Hello.clientId` according to Ethereum conventions.
+    *
+    * Check out examples on https://etcnodes.org
+    *
+    * e.g.
+    * - mantis/v3.0-cd5ae33/linux-amd64/ubuntu-openjdk64bitservervm-java-11.0.9
+    * - besu/v20.10.0/linux-x86_64/oracle_openjdk-java-11
+    * - coregeth/v1.11.8-stable-305b5089/linux-amd64/go1.14.4
+    *
+    * Apparently ethstats expects either 4 parts or 5:
+    * - client/version/os/compiler
+    * - client/identity/version/os/compiler
+    */
+  def nodeName(maybeIdentity: Option[String] = None): String = {
+    val app = {
+      val name = BuildInfo.name
+      val id = maybeIdentity.map("/" + _).getOrElse("") //
```

Comment:
Minor: should we remove the last //?

Reply:
Huh, thanks, I wanted to add a comment but then added it to the docstring.